unikernel.ml

```diff
@@ -1005,313 +1046,425 @@ struct
           ~data:(`String (String.escaped err))
           reqd `Bad_request
 
-  let deploy_form store albatross _ (user : User_model.user) reqd =
+  let delete_albatross_config store albatross_instances _user json_dict reqd =
+    match Utils.Json.get "name" json_dict with
+    | Some (`String name) -> (
+        Store.delete_configuration store name >>= function
+        | Ok _new_configurations ->
+            albatross_instances :=
+              Albatross.Albatross_map.remove name !albatross_instances;
+            Middleware.http_response reqd ~title:"Success"
+              ~data:(`String "Configuration delete successfully") `OK
+        | Error (`Msg err) ->
+            Middleware.http_response reqd ~title:"Error"
+              ~data:(`String (String.escaped err))
+              `Internal_server_error)
```

Comment:
there are two error cases, no? One is that no instance with the given name was found (which should be a bad request), and the other is that store failed (which should be an internal server error, as done here).